app/models/tiebreaker.rb

```diff
@@ -0,0 +1,13 @@
+class Tiebreaker < ActiveRecord::Base
+  enum tiebreaker: [:most_points, :most_weak_side_wins, :fewest_played]
+  validates_uniqueness_of :tiebreaker
```

Comment:
Prefer the new style validations `validates :column, uniqueness: value` over `validates_uniqueness_of`.